alf/examples/main.py

```diff
@@ -77,6 +77,7 @@ def train_eval(train_dir,
         debug_summaries=debug_summaries)
 
 
+@gin.configurable
```

Comment:
Is it possible to avoid having to specify both play.algorithm_ctor train_eval.algorithm_ctor?

Reply:
bind `__main__.play.algorithm_ctor ` with `__main__.train_eval.algorithm_ctor`